portfolio/src/main/webapp/script.js

```diff
@@ -70,3 +70,14 @@ function contact() {
 		</ul>`; 
 	console.log('Contact!'); 
 }
+
+function getComments() {
+	fetch('/data').then(response => response.json()).then((comments) => {
+		console.log(comments);
+		const commentElement = document.body;
```

Comment:
nit - this variable name might be misleading since it's actually the `<body>` element.

Reply:
thanks! updated!